### PR TITLE
Fix broken links [Fixes #2402] 

### LINKED ISSUE #2402: Fix broken links
**Describe the bug**

I came across a few broken links on https://ethereum.org/  across multiple pages.
I ran an analysis test on https://www.deadlinkchecker.com/ and found several links that could be fixed. 
Screen Shot 2021-02-09 at 6 49 42 PM

A list of broken links (found so far): 
- https://careers.labs.consensys.net/
- https://docs.ethhub.io/ethereum-roadmap/layer-2-scaling/optimistic-rollups/
- https://openethereum.github.io/wiki/Warp-Sync-Snapshot-Format
- https://docs.mythx.io/en/latest/
- and more 

**To Reproduce**

1. Go to 'https://ethereum.org/en/community/ '
2. Click on 'Careers at ConsenSys'
3. See error

**Expected behavior**

- All the links should direct to a page with no errors

### PR DESCRIPTION
**Pull request by okdonga**
Fix a few broken links throughout the site. 

## Description

I ran an analysis test on https://www.deadlinkchecker.com/  and found a complete list as below: 

- https://careers.labs.consensys.net/
- https://docs.ethhub.io/ethereum-roadmap/layer-2-scaling/optimistic-rollups/
- https://openethereum.github.io/wiki/Warp-Sync-Snapshot-Format
- https://docs.defisafety.com/audit-process-documentation/process-quality-audit-process
- https://docs.mythx.io/en/latest/
- https://docs.3box.io/build/plugins
- https://www.oasis-open.org/news/pr/baseline-protocol-achieves-key-milestone-with-release-of-v0-1-implementation-for-enterprise-
- https://aleth.io/
- https://https://offchainlabs.com/arbitrum.pdf
- https://www.societegenerale.com/en/newsroom-first-financial-transaction-settled-with-a-digital-currency
- https://aleth.io/
- http://www.finney.org/~hal/rpow/
- http://szabo.best.vwh.net/securetitle.html

Also, I updated the links for  https://ethereum.wiki to https://eth.wiki 

## Related Issue

#2402

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -67,6 +67,8 @@ yarn-error.log
 .pnp.js
 # Yarn Integrity file
 .yarn-integrity
+.yarnrc.yml
+.yarn/
```
Comment (wackerow) [inline .gitignore:71]: Was this intentional? Any chance we could remove?

**Review by wackerow — CHANGES_REQUESTED** [review]
@okdonga Thanks for the help with this! Nice to get these links cleaned up. Just made a small note about the `.gitignore` file, assuming that wasn't intentional. Other than that, looks great! 👍

**Comment by wackerow** [comment]
@all-contributors please add @okdonga for bug reports & content.

File: .gitignore (hunk 1 above)
Comment (okdonga) [reply .gitignore]: Hi @wackerow  Thanks for the review. I have removed the files, but have a question regarding ```.yarn/``` .
When I run ```yarn``` on my local, a bunch of ```.yarn/cache/...``` files are generated. So I added the directory on ```.gitignore``` as I didn't see any past commits on these files. Can you suggest a way to get around this?

Comment (wackerow) [reply .gitignore]: @okdonga I see, not sure why that is happening. @samajammin have you ever had `.yarn/` generated from running `yarn`? Not necessarily against adding it to `.gitignore` just want to make sure there aren't issues with setting up the local environment for some.

Comment (samajammin) [reply .gitignore]: @okdonga - looks like you were able to resolve this? I don't see .gitignore in the changes anymore.

File: src/content/enterprise/index.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ Some collaborative efforts to make Ethereum enterprise friendly have been made b
 
 ### Tooling and libraries {#tooling-and-libraries}
 
-- [Alethio](https://aleth.io/) _Ethereum Data Analytics Platform_
+- [Aleth](https://github.com/ethereum/aleth) _Ethereum Data Analytics Platform_
```
Comment (samajammin) [inline src/content/enterprise/index.md:69]: This is a different project. Aleth is an Ethereum client.

I've pinged Alethio to see if the project is still active: https://twitter.com/samonchain/status/1359307457588207621?s=20

File: .gitignore (hunk 1 above)
Comment (okdonga) [reply .gitignore]: @wackerow , @samajammin 
I think it's due to the **version of yarn** I was using. 
I was using ```2.0.0-rc.27```, and I downgraded to  ```1.22.1``` and I don't see them anymore.  Can I ask what version of yarn you are using? (or suggest using?)

File: src/content/enterprise/index.md (hunk 2 above)
Comment (okdonga) [reply src/content/enterprise/index.md:69]: Thanks for this. I've left a message on their [discord](https://discordapp.com/channels/608940867222372353/608940867222372357/808889052312240128) server too (haven't heard back yet).  However, it seems like https://aleth.io is the official url, even though it's down for now. It's the site listed on their their [twitter](https://twitter.com/AlethioEthstats) profile too. So I will revert the URL back to https://aleth.io/.

**Review by wackerow — DISMISSED** [review]
https://explorer.aleth.io/ Could be used for that Aleth.io link, but other than that this all looks great. Checked all the links, and they were all broken and now work! Thanks again @okdonga

File: .gitignore (hunk 1 above)
Comment (wackerow) [reply .gitignore]: @okdonga I'm using `1.22.10`... the `-rc` is a release candidate version, not on that yet personally

**Review by wackerow — CHANGES_REQUESTED** [review]
@okdonga Mind switching the Aleth.io link to `https://explorer.aleth.io/` in the meantime? That should do it =)

**Review by samajammin — APPROVED** [review]
Approving & merging. Will update Aleth.io links in a separate PR. Thanks again @okdonga!

**Pull request merged**